test/e2e/volume_provisioning.go

```diff
@@ -37,9 +37,14 @@ const (
 	// Expected size of the volume is 2GiB, because all three supported cloud
 	// providers allocate volumes in 1GiB chunks.
 	expectedSize = "2Gi"
+	// Expected size of the externally provisioned volume depends on the external
+	// provisioner: for nfs-provisioner used here, it's equal to requested
+	externalExpectedSize = "1500Mi"
+	// Plugin name of the external provisioner
+	externalPluginName = "example.com/nfs"
 )
 
-func testDynamicProvisioning(client clientset.Interface, claim *v1.PersistentVolumeClaim) {
+func testDynamicProvisioning(client clientset.Interface, claim *v1.PersistentVolumeClaim, external bool) {
```

Comment:
IMO, using `expectedSize` parameter would be more useful than `external bool`. Strictly speaking, fact that all current cloud providers round to 1GB chunks is just a coincidence, it's not a guaranteed property of all internal provisioners.